### PR TITLE
chore(mergify): workflow updates

### PR DESCRIPTION
**Pull request by NGL321**
- Update to reflect issue with automatic dependabot merging
- Addition of label denoting core team member PRs

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # See https://doc.mergify.io
 
 pull_request_rules:
+  - name: label core
+    actions:
+      label:
+        add: [ contribution/core ]
+    conditions:
+      - author=eladb
+      - author=RomainMuller
```
Comment (eladb) [inline .mergify.yml]: Are you sure this works? I think the conditions are combined through an AND operation and not through OR.

Comment (eladb) [reply .mergify.yml]: Also replace the condition below that lists all the owners with the label. We don’t need this list twice.

Comment (RomainMuller) [reply .mergify.yml]: OMG you're right.

And w/r/t the label... Have to be 100% certain that nobody off the list can assign labels.

Comment (NGL321) [reply .mergify.yml]: Ahh, I didnt realize this

**Pull request merged**